### PR TITLE
Add SIP support for multiple domains

### PR DESCRIPTION
**Pull request by msach22**

### CODE REVIEW

**Comment by jeffswartz** [comment]
Please add a new env config variable: `GOOGLE_HOSTED_DOMAINS` (`Google.hostedDomains` in the config.json file), and document it in the README. Maintain backward compatibility for the old setting (`GOOGLE_HOSTED_DOMAIN`, singular), but leave it undocumented.

File: views/room.ejs [hunk 1]
```diff
@@ -319,7 +319,7 @@
           <div class="google-sign-in-note-container">
             <div class="google-sign-in-note">
               <span class=title>Note:</span><br/>
-              <span>This function requires you to sign in with your <b>@tokbox.com</b> Google account</span>
+              <span>This function requires you to sign in with your <b>@tokbox.com</b> or <b>@vonage.com</b> Google account</span>
```
Comment (jeffswartz) [inline views/room.ejs]: Please generate this string programmatically, based on the domain name(s). Do not hard-code in `@tokbox.com` and `@vonage.com`.

File: web/js/roomController.js [hunk 2]
```diff
@@ -824,7 +824,7 @@ RecordingsController, ScreenShareController, FeedbackController, PhoneNumberCont
     _allHandlers = RoomStatus.init(_allHandlers, { room: _sharedStatus });
 
     if (enableSip && requireGoogleAuth) {
-      GoogleAuth.init(aParams.googleId, aParams.googleHostedDomain, function (aGoogleAuth) {
+      GoogleAuth.init(aParams.googleId, null, function (aGoogleAuth) {
```
Comment (jeffswartz) [inline web/js/roomController.js]: Should we remove the second parameter of `GoogleAuth.init()`? (Is it always set to none?)

**Comment by msach22** [comment]
Closing b/c feature is not needed anymore

**Pull request closed without merging**